### ISSUE TITLE
Long mutual fund folio scheme name is not fully read

### ISSUE THREAD

**Issue by prasanthxyz** [codereverser/casparser#73]
For long mutual fund scheme names that spans more than one row, only the first row is being read.
Example name:
"""
My long mutual fund scheme name ELSS -
Direct growth plan
"""
Only first row will be read: "My long mutual fund scheme name ELSS -"

**Comment by codereverser** [comment]
Can you please send me a partial screenshot of the entire two rows (with all sensitive data hidden) just to get an idea about how it looks? 

Also does the remaining half just get ignored or does it appear in other place (like appended to the next schema name etc)?

**Comment by prasanthxyz** [comment]
Hi,

Sure, attaching the partial screenshot of input and output entire rows -- check **only first two rows** in the screenshot.
![input](https://github.com/codereverser/casparser/assets/1196708/099ee925-fd80-482a-86c9-4b18e690dc1e)
![output](https://github.com/codereverser/casparser/assets/1196708/393c54e6-3d81-4785-9454-2a0d6447e176)

As far as I found, the remaining half just get ignored -- didn't find it as part of any other entry.

I tried to fix it for a while but failed and ended up manually copy pasting because only 3 such entries for me, but would be nice to automate this.
